CodeGeneration/Sources/SyntaxSupport/DeclNodes.swift

```diff
@@ -1460,7 +1471,8 @@ public let DECL_NODES: [Node] = [
         documentation: "The `#` sign."
       ),
       Child(
-        name: "Macro",
+        name: "MacroName",
```

Comment:
We use `Name` for most other nodes, any reason to prefer `MacroName` here?

Reply:
We use `Name` for nodes that declare something (e.g. `StructDeclSyntax`), while this node refers to something. And I just thought that name alone here is a little ambiguous because it’s not entirely sure what the name is. I don’t have super strong opinions here though.